### PR TITLE
Some trivial cleanups of BigQuery tests

### PR DESCRIPTION
**Pull request by jkff**
- Split the test in 3 files
- In write test, put some common logic into rules and member variables
- In write test, removed trivial builder tests

### CODE REVIEW

File: sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOWriteTest.java [hunk 1]
```diff
@@ -0,0 +1,1379 @@
...
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.TemporaryFolder;
+import org.junit.rules.TestRule;
+import org.junit.runner.Description;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.junit.runners.model.Statement;
+
```
Comment (chamikaramj) [inline sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOWriteTest.java]: Please add a doc comment.

File: sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOWriteTest.java [hunk 2]
```diff
@@ -0,0 +1,1379 @@
...
+        @Override
+        public void evaluate() throws Throwable {
+          options = TestPipeline.testingPipelineOptions();
+          options.as(BigQueryOptions.class).setProject("project-id");
+          options.as(BigQueryOptions.class).setTempLocation(testFolder.getRoot().getAbsolutePath());
+          p = TestPipeline.fromOptions(options);
+          p.apply(base, description).evaluate();
+        }
+      };
+      return testFolder.apply(withPipeline, description);
```
Comment (chamikaramj) [inline sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOWriteTest.java]: Why do we invoke 'testFolder' here ? Worth adding a comment to clarify ?

File: sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOReadTest.java [hunk 3]
```diff
@@ -0,0 +1,831 @@
...
+import org.apache.beam.sdk.values.PBegin;
+import org.apache.beam.sdk.values.PCollection;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
```
Comment (chamikaramj) [inline sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOReadTest.java]: Please add a doc comment.

File: sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryHelpersTest.java [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+import org.apache.beam.sdk.transforms.windowing.IntervalWindow;
+import org.apache.beam.sdk.util.CoderUtils;
+import org.apache.beam.sdk.util.WindowedValue;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
```
Comment (chamikaramj) [inline sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryHelpersTest.java]: Please add a doc comment.

**Comment by jkff** [comment]
All done, thanks. Merging.

**Pull request merged**